packages/notifi-react/lib/context/NotifiTopicContext.tsx

```diff
@@ -115,10 +116,21 @@ export const NotifiTopicContextProvider: FC<PropsWithChildren> = ({
   ) => {
     const createAlertInputs: Types.CreateFusionAlertInput[] = [];
     topicWithFilterOptionsList.forEach(
-      ({ topic, filterOptions, subscriptionValue, customAlertName }) => {
+      ({ topic, filterOptions, customAlertName, subscriptionValue }) => {
         const fusionEventId = topic.fusionEventDescriptor.id;
         const fusionEventMetadata = getFusionEventMetadata(topic);
         if (!fusionEventMetadata || !fusionEventId) return;
+        let subscriptionValueObject: InputObject[] | null = [
+          { label: '', value: '' },
+        ];
+        if (!subscriptionValue) {
+          const subscriptionValueOrRef =
+            getFusionEventMetadata(topic)?.uiConfigOverride
+              ?.subscriptionValueOrRef;
+          subscriptionValueObject = subscriptionValueOrRef
+            ? [...resolveObjectArrayRef('', subscriptionValueOrRef, inputs)]
+            : null;
+        }
```

Comment:
LGTM, just a small suggestion, how do you think.

```suggestion
            : null;
        }
        subscriptionValue = subscriptionValueObject?.[0]?.value;
```